public_html/assets/css/styles.css

```diff
@@ -78,15 +78,11 @@ input:focus {background-color:transparent;}
 /* ---------------------------------
     HOME BG (z-index: -13, -12, -11)
 ------------------------------------ */
-.bg-images {position:fixed; top:0; left:0; width:100%; height:100%; z-index:-20;}
-.bg-img {position:absolute; top:0; left:0; bottom:0; width:100%; background:#fff none center center no-repeat; z-index:-13; background-size:contain;}
+.bg-images {position:fixed; top:0; left:0; width:100%; height:100%; z-index:-20; background-color:#fff;}
+.bg-img {position:absolute; top:0; right:0; bottom:0; width:52.5%; z-index:-13; background:transparent none center center no-repeat; background-size:cover;}
```

Comment:
width is just barely non-centered, which is similar-ish to the slight rectangular images.